identity-service/src/notifications/index.js

```diff
@@ -230,6 +246,24 @@ class NotificationProcessor {
       done(error)
     })
 
+    // Download Email notification queue
+    this.downloadEmailQueue.process(async (job, done) => {
+      logger.info('processDownloadEmails')
+      let error = null
+      try {
+        await processDownloadAppEmail(expressApp, audiusLibs)
+        await this.redis.set(NOTIFICATION_DOWNLOAD_EMAIL_JOB_LAST_SUCCESS_KEY, new Date().toISOString())
+      } catch (e) {
+        error = e
+        logger.error(`processDownloadEmails - Problem with processing emails: ${e}`)
+        this.errorHandler(e)
+      }
+      // Wait 10 minutes before re-running the job
```

Comment:
why do we want to wait 10 minutes in either job

Reply:
we can remove - I don't think there is a need